libs/picomodel/pm_fm.c

```diff
@@ -210,7 +210,7 @@ static picoModel_t *_fm_load( PM_PARAMS_LOAD ){
 	fm_xyz_st_t     *triangle;
 	fm_frame_t      *frame;
 
-	picoByte_t      *bb, bb0;
+	picoByte_t      *bb, *bb0;
 	picoModel_t *picoModel;
```

Comment:
This seems like an obvious error. The incorrect storage type of this variable was causing 11 or so warnings that went away when I converted it to a pointer. This variable is passed to the Pico free function, so I'm fairly sure that's correct.